setup.py

```diff
@@ -49,9 +49,7 @@ def requirements(file):
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
     install_requires=requirements("requirements.txt"),
-    extras_require={
-        "develop": requirements("requirements_dev.txt"),
-    },
+    extras_require={"develop": requirements("requirements_dev.txt"), "rdkit": ["rdkit-pypi"]},
```

Comment:
@thinkh : works as intended / specified. If you don't have rdkit, you don't get the rdkit-endpoints. If you install tdp-core[rdkit] (it's called an extras / a package flavour in other languages), you get it.

Imo our project structure doesn't perfectly support the concept of optional packages and I think it could be a nice standalone tdp package. (However: Complexity of yet another package)
Or we just unconditionally take it into tdp_core, it's just 20mb of mostly binaries after all.

Reply:
Ok, thanks for the clarification. Do you know how I install `tdp-core[rdkit]` as part of the `docker-compose build api` (or `docker-compose up`)? We would need to call this somehow later when building the Docker images for the deployment.

Reply:
Assuming the generator isn't messing it up, just add
replace `-e /phovea/tdp_core` by `-e /phovea/tdp_core[rdkit]` in the requirements_workspaces.txt or to the product like this:
https://github.com/datavisyn/ordino_reprovisyn_product/pull/13/files (testing that currently in progress)

Reply:
Checkout/build seems to work. But I'm having some port conflict so I can't docker-compose-up.

Regarding the default-title:
To my understanding it should be a tag, right? I added that.

Reply:
As I said, I think we should remove the flavor (as it isn't really supported by our infra right now) and just add it to the dependencies. In the end, it doesn't matter if the server image is 20MB more or less. 
@dvmarkusvogl could you do this? Then you also don't need to check for the availability of the module. 
@thinkh  do you agree?

Reply:
Yes, I agree.

Reply:
Ok, i made rdkit mandatory (by now you don't even need rdkit-pypi, the main rdkit is now the same, suggested package)

I also deleted some code for handling a lack of rdkit